etlman/projects/forms.py

```diff
@@ -46,3 +46,21 @@ class Meta:
                 attrs={"onchange": "selectedLanguage(this.value);"}
             ),
         }
+
+
+class PipelineScheduleForm(forms.ModelForm):
+    class Meta:
+        model = PipelineSchedule
+        fields = [
```

Comment:
I would like to set placeholder rather than help_text in some form fields, but based on my search Django does not support placeholder text.

Reply:
Tobias and Janelle's eyes helped figure this out.